home/src/app/pages/goods/goods.html

```diff
@@ -121,4 +121,42 @@
 			</ul>
 		</div>
 	</div>
-</div>	
\ No newline at end of file
+
+
+	<!-- Modal -->
+	<div id="exchangeModal" class="modal animated fadeIn" role="dialog">
```

Comment:
@greenlucky are implementing this feature, work with mail box & exactly location.